package.json

```diff
@@ -50,5 +51,18 @@
     "exclude": [
       "spec"
     ]
+  },
+  "xo": {
+    "space": 2,
+    "rules": {
+      "space-before-function-paren": ["error", "never"]
+    },
+    "globals": [
+      "describe",
```

Comment:
You can use:
```json
{
  "env": {
    "jasmine": true
  }
}
```
instead of adding every global one by one

Reply:
Damn, didn't know about this... Super useful 👍

Reply:
FYI https://eslint.org/docs/user-guide/configuring#specifying-environments